front/app/containers/HomePage/SignedInHeader/FallbackStep.tsx

```diff
@@ -68,7 +67,7 @@ const FallbackStep = ({ currentOnboardingCampaignName }: Props) => {
       >
         <HeaderContent id="e2e-signed-in-header-default-cta">
           <Left>
-            {defaultMessage && !isEmpty(defaultMessage) ? (
+            {defaultMessage && !isEmptyMultiloc(defaultMessage) ? (
```

Comment:
I find defaultMessage a confusing var name. I'd rather do `signedInHeaderMultiloc`. The default message is the fallback message.